src/components/dialog/dialog.tsx

```diff
@@ -9,7 +10,7 @@ import {
     Prop,
     Watch,
 } from '@stencil/core';
-import * as focusTrap from 'focus-trap';
+// import * as focusTrap from 'focus-trap';
```

Comment:
Opening the dialog shows the error below in the dev tool. I can see that you re trying to import FocusTrap here. 

<img width="482" alt="Screenshot 2021-07-15 at 11 29 33" src="https://user-images.githubusercontent.com/22968497/125765383-c250e63f-24a8-41eb-853b-20d82e213e58.png">

Reply:
Ah, yeah, good find! I commented out the importing of focusTrap, because it caused a build error. I think the new dialog had some updated functionality for focus trapping…

Reply:
I'll have a look at this straight away.

Reply:
Oh yeah! 🤘

![image](https://user-images.githubusercontent.com/154725/125776724-9f0046b9-1e6f-4f58-9c8f-3008a6821883.png)

Reply:
⚡